doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+The Things Industries\' [command-line interface]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. The objective with the introduction of telemetry is to be able to better understand what is used so that we can expand and improve on the experience that we provide to our users.
+
+## Identification
+
+The worry that data is attached to an easily identifiable piece of information is common and for good reason, therefore we believe that it is the first thing we should address.
+
+When the CLI start the procedure to collect and send the telemetry data for the first time, it creates a file on the system's cache folder which contains the following data: 
+```yaml
+uid: 00000000-0000-0000-0000-000000000000
+last_sent: 2023-08-02T16:45:53.742417-03:00
+```
+
+The `uid` field attracts the most attention since initally an user does not know how its value is generated, but to set its value we use Google's [golang package](https://github.com/google/uuid) which is based on the [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122) and DCE 1.1: Authentication and Security Services.
+
+## Types of telemetry data
+
+Bellow is an JSON object with an example of a full telemetry data message that is sent from the CLI to us:
+
+```json
+{
+  "uid": "00000000-0000-0000-0000-000000000000",
+  "os": {
+    "operating_system": "linux",
+    "arch": "amd64",
+    "binary_version": "3.27.0-dev",
+    "golang_version": " go1.20.7"
+  },
+  "cli": {},// An empty object instead of `null` to represent that the data is sent by a CLI.,
+  "entities_count": null
+}
+```
+
+{{< note >}}
+All new data collected will be added bellow with the respective version in which the field was introduced. 
+{{</ note >}}
+
+List of fields and its purpose on by version:
+
+{{< new-in-version "3.27.1">}}
```

Comment:
This whole functionality is new to that version.

Reply:
Can you explain why is it important for users to know which fields were introduced by which TTS version? Because usually we delete these new-in-version tags after 2-3 minor versions

Reply:
well the intent behind this was to have transparency in regards to what was added when due to the possibility of users having different versions of the Stack in their own deployment. But considering that each new telemetry data collected can be disabled separately its probably okay to not specify the version in which they were introduced.